test_args.sh

```diff
@@ -19,7 +19,7 @@ cat > $TMPDIR/key <<ETH_KEY
 ETH_KEY
 
 ETH_ARGS="-ethKeystorePath $TMPDIR/key"
-SLEEP=0.25
+SLEEP=0.5
```

Comment:
Since we're not adding any additional tests in this file, can we just revert all changes? Are any of these changes necessary?

Reply:
Oh sorry I forgot to change `SLEEP` back

We need the re-assignment of `CUSTOM_DATA_DIR` else we'll get an error saying that we're expecting a different chain ID